Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -384,12 +387,21 @@ private void PrepareTest(string compressionType, StageType stageType, string sta
             t_stageType = stageType;
             t_compressionType = compressionType;
             t_autoCompress = autoCompress;
-            
             // Prepare temp file name with specified file extension
             t_fileName = Guid.NewGuid() + ".csv" + 
                         (t_autoCompress? SFFileCompressionTypes.LookUpByName(t_compressionType).FileExtension: "");
             t_inputFilePath = Path.GetTempPath() + t_fileName;
-            t_outputFilePath = $@"{s_outputDirectory}/{t_fileName}";
+            if (UseGzipCompressionForNotCompressedFilesWithAutoCompressOption())
```

Comment:
this UseGzip.. method is not using any variables of PrepareTest. I'd change it

Reply:
this method is now IsCompressedByTheDriver()